django_slack/utils.py

```diff
@@ -10,14 +10,14 @@ class Backend(object):
     def send(self, url, data):
         raise NotImplementedError()
 
-    def validate(self, content_type, content):
+    def validate(self, content_type, content, original_data):
```

Comment:
Called ``original_data`` here but ``data`` in ``SlackException`` and in ``RequestsBackend``. Deliberate?